Neurotrauma/Xml/items.xml

```diff
@@ -2282,6 +2282,9 @@
       <!-- making the organ go kaputt if left outside of refrigeration -->
       <StatusEffect type="Always" target="This" condition="-0.5">
         <Conditional hastag="neq refrigerated" targetcontainer="true" />
+      </StatusEffect>
+	  <StatusEffect type="Always" target="This" condition="0.5" conditionalComparison="And" >
```

Comment:
i don't know why git display the green lines in the wrong place, the code should be fine